app/javascript/styles/mastodon/variables.scss

```diff
@@ -36,6 +36,8 @@ $dark-text-color: $ui-base-lighter-color !default;
 $secondary-text-color: $ui-secondary-color !default;
 $highlight-text-color: $ui-highlight-color !default;
 $action-button-color: $ui-base-lighter-color !default;
+$passive-text-color: #ccaa00 !default;
```

Comment:
I suggest using existing colors like success-green and gold-star to keep the color scheme consistent

Reply:
Fix it.